CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 - Moved character counter to bottom of multiline text input ([#992](https://github.com/Shopify/polaris-react/pull/992))
 - Aligned `TopBar` search input and results with page content ([#1008](https://github.com/Shopify/polaris-react/issues/1008))
+- Fixed an issue on Chrome when you use a `TextField` inside `Collapsible` which is inside a scrollable element, the text disappeared if you focused a fully hidden `TextField` ([#1047](https://github.com/Shopify/polaris-react/pull/1047))
```

Comment:
We use the `UNRELEASED.md` file for entries (and then move them over during a release).